.eslintignore

```diff
@@ -0,0 +1,2 @@
+examples
+scripts
```

Comment:
File should end with new line (see [this stackoverflow question](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline)